backend/transcribee_backend/config.py

```diff
@@ -7,6 +7,7 @@ class Settings(BaseSettings):
     storage_path: Path = Path("storage/")
     secret_key = "insecure-secret-key"
     worker_timeout = 60  # in seconds
+    tokens_timeout = 60 * 24  # in minutes, once per day
```

Comment:
IMHO we can just hardcode a time in `setup_periodic_tasks`. I think this timeout should be 1h at most